core/controllers/classifier.py

```diff
@@ -121,27 +149,30 @@ def post(self):
         return self.render_json({})
 
     @acl_decorators.open_access
-    def get(self):
+    def get(self) -> None:
         """Handles GET requests.
 
         Retrieves the name of the file on GCS storing the trained model
         parameters and transfers it to the frontend.
         """
-        exploration_id = self.normalized_request.get('exploration_id')
-        state_name = self.normalized_request.get('state_name')
+        assert self.normalized_request is not None
+        exploration_id = self.normalized_request['exploration_id']
+        state_name = self.normalized_request['state_name']
 
         try:
-            exp_version = int(self.normalized_request.get(
-                'exploration_version'))
+            exp_version = int(self.normalized_request['exploration_version'])
             exploration = exp_fetchers.get_exploration_by_id(
                 exploration_id, version=exp_version)
             interaction_id = exploration.states[state_name].interaction.id
         except Exception as e:
             raise self.InvalidInputException(
                 'Entity for exploration with id %s, version %s and state %s '
                 'not found.' % (
-                    exploration_id, self.normalized_request.get(
-                        'exploration_version'), state_name)) from e
+                        exploration_id,
```

Comment:
Deindent this and the following few line by 4. (The closing paren should be at the same indentation level as the line with the opening paren.)

Reply:
Done indentation is fixed